integration/integration_linux_test.go

```diff
@@ -145,9 +154,13 @@ var _ = Describe("Basic PTP using cnitool", func() {
 
 			chainedBridgeBandwidthEnv.runInNS(hostNS, cnitoolBin, "del", "network-chain-test", contNS1.LongName())
 			basicBridgeEnv.runInNS(hostNS, cnitoolBin, "del", "network-chain-test", contNS2.LongName())
+
+			contNS1.Del()
```

Comment:
We should do this from an AfterEach() for this specific testcase, because otherwise if ther'es an error during the test, these functions at the end won't run. AfterEach() will catch them even if the test block fails though.

Reply:
@dcbw But it's already in an AfterEach clause no ? Unless I don't understand your point :)

Reply:
https://github.com/containernetworking/plugins/pull/921/files#diff-5f864559e3f42ccd0717a6fb1c6b1a99f5e88d926ef659d0db8f4038673fed2eR147